powersimdata/network/usa_tamu/constants/zones.py

```diff
@@ -85,7 +85,11 @@
     | interconnect2loadzone["Western"]
     | interconnect2loadzone["Texas"]
 )
-
+# Map interconnect to load zone id
+interconnect2id = {
+    k: sorted([zone_df.reset_index().set_index("zone_name").loc[l].zone_id for l in v])
```

Comment:
Is it equivalent to `sorted([zone_df[zone_df.zone_name == l].index for l in v])`?

Reply:
I simplified it further:
```
interconnect2id = {
    k: set(zone_df.isin(v).query("zone_name == True").index)
    for k, v in interconnect2loadzone.items()
}
```

Reply:
By the way, I ended up not using this `dict` since @danielolsen came up with a different way to obtain the mapping in the `TransformProfile` class saving an import. I leave it since it may come up handy in the future.

Reply:
Nice.